### PR TITLE
WIP remove duplicate files and update links

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**What**
Making some updates to the Community Docs on aerogear.org

**Why**
These are out of date

**Changes**
_Commit 1_ 
- [x] Remove duplicate copies of contributing guides - the files removed most recent update is 8 months ago and some go back 3 years. 
- [x] Update link to more recently updated versions of these files - The ones kept are more recently updated and will be updated as part of this pr
- [x] Remove Code of Conduct Link - it doesn't go anywere

_Commit 2_
- [x] Update `Contributing to AeroGear`

_Commit 3_
- [x] Update `AeroGear GitHub Workflow`

### CODE REVIEW

File: community/contributingGuides/Contributing.asciidoc [hunk 1]
```diff
@@ -15,15 +15,15 @@ As an open source project we are always on the lookout for community members tha
 
 === Apache License
 
-All contributions to AeroGear are made under the http://apache.org/licenses/LICENSE-2.0.txt[Apache License, Version 2.0].
+All contributions to AeroGear are made under the http://apache.org/licenses/LICENSE-2.0.txt[Apache License, Version 2.0^].
```
Comment (pb82) [inline community/contributingGuides/Contributing.asciidoc]: Is the `^` intentional? What does it mean?

File: community/contributingGuides/Contributing.asciidoc [hunk 2]
```diff
@@ -15,15 +15,15 @@ As an open source project we are always on the lookout for community members tha
...
 
 ==== Found An Issue?
-Have you run into an issue with one of our examples, sites, or have a suggestion? The best place to start is probably talking about it in the forums or IRC. If you've already done that, or just want to create the issue please do at https://issues.jboss.org/browse/AEROGEAR.
+Have you run into an issue with one of our https://docs.aerogear.org/aerogear/latest/con_services.html[services], https://github.com/aerogear?utf8=%E2%9C%93&q=showcase&type=&language=[apps^], or link:../../../sdks[sdks] or just have a suggestion? The best place to start is probably talking about it in the google group or IRC. If you've already done that, or just want to create the issue please do in the https://issues.jboss.org/browse/AEROGEAR[AEROGEAR project on JIRA^].
```
Comment (pb82) [inline community/contributingGuides/Contributing.asciidoc]: We could also mention that they are welcome to open Github issues.

File: community/contributingGuides/Contributing.asciidoc (hunk 1 above)
Comment (laurafitzgerald) [reply community/contributingGuides/Contributing.asciidoc]: @pb82 it opens a new tab. I used it anywhere there is a link to something external from the site.

File: community/contributingGuides/Contributing.asciidoc (hunk 2 above)
Comment (laurafitzgerald) [reply community/contributingGuides/Contributing.asciidoc]: @pb82 will do

File: community/contributingGuides/Contributing.asciidoc (hunk 1 above)
Comment (pb82) [reply community/contributingGuides/Contributing.asciidoc]: Thanks, didn't know that

**Comment by laurafitzgerald** [comment]
@pb82 thanks for the review, I pushed the change you suggested. Would you mind taking another look?

**Pull request merged**